src/main/java/cubicchunks/world/ICubicWorld.java

```diff
@@ -139,7 +140,7 @@ default boolean testForCubeAndNeighbor(Cube cube, @Nullable Predicate<Cube> test
 
 	int getEffectiveHeight(int blockX, int blockZ);
 
-	//this is a hack
+	//this is a hack TODO: remove!
 	void setGeneratingWorld(boolean generating);
```

Comment:
This hack was my attempt to force light updates when generating cubes. It's not used anymore.